### ISSUE TITLE
runTest is not honoring the launchArg '--new-window'

### ISSUE THREAD

**Issue by lovettchris** [microsoft/vscode#144115]

**Comment by lovettchris** [comment]
@connor4312 - I don't think so.  When I manually run "code --new-window" I always get a new empty window, it never restores previously opened files in that mode.  So why is the vscode-test doing something different than that?

**Comment by connor4312** [comment]
You may have a user setting that configures that. `vscode-test` runs in a fresh user data dir by default.

**Comment by lovettchris** [comment]
Interesting, I'm definitely NOT getting a fresh user-data on each call to `runTest(...)` which is why I had to implement my own `clearUserWorkspaceData` function.  See [my new unit test](https://github.com/lovettchris/vscode-lean4/tree/clovett/unittest/vscode-lean4/test) that uses `@vscode/test-electron`

**Comment by connor4312** [comment]
Sorry, I meant that the user data dir for tests is separate from the standard user data dir (usually in `~/.code` or `~/.code-insiders`), so any settings you have in that profile that might effect it won't apply to the test profile.

**Comment by lovettchris** [comment]
Yes that is true, and the .vscode-test folder is created and a test version of vscode is installed, and that's all great, it's just the state across separate `runTests` that doesn't seem to be honoring the `--new-window` arg.  You can see I'm passing that arg like this:

```typescript
await runTests({
	vscodeExecutablePath,
	extensionDevelopmentPath,
	extensionTestsPath: path.resolve(__dirname, 'lean3'),
	launchArgs: ['--new-window', '--disable-gpu', testFolder] });
```
but alas, it's not working.
